src/test/java/net/sf/jabref/bibtex/BibEntryWriterTest.java

```diff
@@ -53,24 +51,28 @@ public void testSerialization() throws IOException {
 
         String actual = stringWriter.toString();
 
+        // @formatter:off
         String expected = Globals.NEWLINE + Globals.NEWLINE + "@Article{," + Globals.NEWLINE +
-                "  Author                   = {Foo Bar}," + Globals.NEWLINE +
-                "  Journal                  = {International Journal of Something}," + Globals.NEWLINE +
-                "  Note                     = {some note}," + Globals.NEWLINE +
-                "  Number                   = {1}" + Globals.NEWLINE +
+                "  author                   = {Foo Bar}," + Globals.NEWLINE +
+                "  journal                  = {International Journal of Something}," + Globals.NEWLINE +
+                "  note                     = {some note}," + Globals.NEWLINE +
+                "  number                   = {1}" + Globals.NEWLINE +
                 "}";
+        // @formatter:on
 
         assertEquals(expected, actual);
     }
 
     @Test
     public void roundTripTest() throws IOException {
+        // @formatter:off
         String bibtexEntry = "@Article{test," + Globals.NEWLINE +
-                "  Author                   = {Foo Bar}," + Globals.NEWLINE +
-                "  Journal                  = {International Journal of Something}," + Globals.NEWLINE +
-                "  Note                     = {some note}," + Globals.NEWLINE +
-                "  Number                   = {1}" + Globals.NEWLINE +
+                "  author                   = {Foo Bar}," + Globals.NEWLINE +
+                "  journal                  = {International Journal of Something}," + Globals.NEWLINE +
+                "  note                     = {some note}," + Globals.NEWLINE +
+                "  number                   = {1}" + Globals.NEWLINE +
                 "}";
+        // @formatter:on
 
```

Comment:
Please do not simply rewrite the tests this way! I used this test to ensure that a rewrite without modification does not change the serialization. Your change eliminates this information! Please only change the test that modify an entry (which results in a reformatting on write)